pom.xml

```diff
@@ -2,36 +2,79 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.chariotsolutions</groupId>
-    <artifactId>meetup-notifier</artifactId>
-    <version>1.0-SNAPSHOT</version>
+  <groupId>com.chariotsolutions</groupId>
+  <artifactId>meetup-notifier</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
@kyleswartley Actually we should the version 0.1-SNAPSHOT - I should have caught that earlier.